platform/os_windows.go

```diff
@@ -302,32 +303,101 @@ func restartHNS(ctx context.Context) error {
 	}
 	defer service.Close()
 	// Stop the service
-	_, err = service.Control(svc.Stop)
-	if err != nil {
-		return errors.Wrap(err, "could not stop service")
+	log.Printf("Stopping HNS service")
+	_ = retry.Do(
```

Comment:
would exponential back off make sense here?  if its not comign up should we wait longer.